src/SUMMARY.md

```diff
@@ -5,9 +5,8 @@
   - [Using the Standard Library (`std`)](./overview/using-the-standard-library.md)
   - [Bare Metal (`no_std`)](./overview/bare-metal.md)
   - [Comparing `std` and `no_std`](./overview/comparing-std-and-no_std.md)
-- [Installation](./installation/index.md)
-  - [Rust on Espressif SoCs](installation/rust-on-esp.md)
-  - [Installing Rust for Espressif SoCs](./installation/installing-rust-for-esp.md)
+- [Rust on ESP targets](./installation/index.md)
+  - [Installation](./installation/installing-rust-for-esp.md)
```

Comment:
Feel free to disagree, but I think we should probably try to keep the menu items' text in line with the titles of each respective page.